.github/workflows/check.yml

```diff
@@ -121,10 +139,21 @@ jobs:
         apt-get update -y
         apt-get install -y software-properties-common
         add-apt-repository ppa:alexlarsson/flatpak
-        apt-get install -y automake autoconf libtool gettext autopoint gcc \
-          gtk-doc-tools shared-mime-info desktop-file-utils \
-          libglib2.0-dev libjson-glib-dev libfontconfig1-dev \
-          libgdk-pixbuf2.0-dev libsystemd-dev
+        apt-get install -y \
+          automake \
+          autoconf \
+          libtool \
+          gettext \
+          autopoint \
+          gcc \
+          gtk-doc-tools \
+          shared-mime-info \
+          desktop-file-utils \
+          libglib2.0-dev \
+          libjson-glib-dev \
+          libfontconfig1-dev \
+          libgdk-pixbuf2.0-dev \
+          libsystemd-dev
```

Comment:
at the risk of being unnecessarily annoying, but this would be nice as sorted list - the one above is but this one isn't